src/LibDeclarations/webxr.d.ts

```diff
@@ -315,11 +315,16 @@ interface XRJointPose extends XRPose {
     radius: number | undefined;
 }
 
+// to be extended
+type XRHandJoint = string;
+
 interface XRHand extends Iterable<XRJointSpace> {

```

Comment:
XRHand is not an array anymore; instead it's a map: `iterable<XRHandJoint, XRJointSpace>`

Reply:
This PR is to first get the hands working again.
I am already implementing a new version using the `fillXXX` functions to create less objects per frame. 
XRHandJoint is technically not a string but a collection  of predefined strings. To avoid back-compat issues i didn't change it yet, but this is coming together with the new PR as well.